src/report/repository/report-repository.service.ts

```diff
@@ -7,21 +7,15 @@ import {
 import { HttpService } from '@nestjs/axios';
 import { ConfigService } from '@nestjs/config';
 import { catchError, map, Observable } from 'rxjs';
-
-interface ReportEntity {
-  id: string;
-  key: string;
-  value: {
-    rev: string;
-  };
-  doc: ReportDoc;
-}
+import { CouchDbRow } from '../../couchdb/dtos';
 
 interface ReportDoc {
   _id: string;
   _rev: string;
   title: string;
-  aggregationDefinitions: any; // TODO better typing
+  aggregationDefinitions: {
+    fields: string[];
+  };
```

Comment:
is this correct? In the sample Reports I have seen this just as `aggregationDefinitions: [ "SELECT * FROM X"]`

Reply:
ah cool, that's true. How about naming this different than "aggregationDefinitions" then, to avoid confusion for people drawing hasty conclusions, like me? :sweat_smile: